samigo/samigo-app/src/webapp/jsf/author/editPart.jsp

```diff
@@ -53,22 +53,11 @@
           <!--Initialize bootstrap multiselect-->
           $("#modifyPartForm\\:multiplePools").attr("multiple", "multiple");
 
-          var divElem = document.createElement('div');
-          var filterPlaceholder = <h:outputText value="'#{assessmentSettingsMessages.multiselect_filterPlaceholder}'" />;
-          divElem.innerHTML = filterPlaceholder;
-          filterPlaceholder = divElem.textContent;
-          var selectAllText = <h:outputText value="'#{assessmentSettingsMessages.select_all_groups}'" />;
-          divElem.innerHTML = selectAllText;
-          selectAllText = divElem.textContent;
-          var nonSelectedText = <h:outputText value="'#{assessmentSettingsMessages.multiselect_nonSelectedText}'" />;
-          divElem.innerHTML = nonSelectedText;
-          nonSelectedText = divElem.textContent;
-          var allSelectedText = <h:outputText value="'#{assessmentSettingsMessages.multiselect_allSelectedText}'" />;
-          divElem.innerHTML = allSelectedText;
-          allSelectedText = divElem.textContent;
-          var nSelectedText = <h:outputText value="'#{assessmentSettingsMessages.multiselect_nSelectedText}'" />;
-          divElem.innerHTML = nSelectedText;
-          nSelectedText = divElem.textContent;
+          const filterPlaceholder = <h:outputText value="'#{assessmentSettingsMessages.multiselect_filterPlaceholder}'" />;
```

Comment:
Better use template literals using backtick instead of apostrophe, we can have JS errors.

https://github.com/sakaiproject/sakai/pull/12179/files#diff-c7b5cea01bf30ebeb1ee4bc8c6b834dd0128ea111bfc9e130bedb31a29a55789R80